languages/en.php

```diff
@@ -148,6 +148,7 @@
 	'access:read' => "Read access",
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
+	'access:missing_name' => "Missing access level name",
```

Comment:
uh you added it: https://github.com/Elgg/Elgg/commit/437f9649a9204db632cae655bc88f6fbf0880b1c this PR only fixes that you missed it adding it to the translation file :)